OpenChange/MAPIStoreMailMessage.m

```diff
@@ -537,6 +537,17 @@ - (void) _fetchBodyData
     }
 
   bodySetup = YES;
+  /* Free data from header setup that it is only used for setup the body */
+  [bodyContentKeys release];
+  bodyContentKeys = nil;  
```

Comment:
It should be set to nil because if not we can have a double release. DESTROY just does the same than this code, dealloc then set to nil. So we can use it you feel is neater.